assets/src/application/ui/input/EspressoButton/EspressoIconButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { IconButton, IconButtonProps } from '@infraUI/inputs';
+import { EspressoIconButtonProps } from './types';
+import { withLabel, withTooltip } from '../../display';
+
+type ButtonType = React.ComponentType<EspressoIconButtonProps>;
+
+const EspressoIconButton = React.forwardRef<typeof IconButton, IconButtonProps>(
```

Comment:
let's name it just `IconButton`
we had a convo about `Espresso` prefix in slack last week and @tn3rb agreed that we can go with a shorter version w/o the prefix

Reply:
We already have `IconButton` in infrastructure folder.